src/Kotlin/KotlinLanguageService.kt

```diff
@@ -171,24 +164,46 @@ class KotlinLanguageService : LanguageService {
         }
     }
 
+    private fun ContentNode.renderAnnotationsForNode(node: DocumentationNode) {
+        node.annotations.forEach {
+            renderAnnotation(it)
+        }
+    }
+
+    private fun ContentNode.renderAnnotation(node: DocumentationNode) {
```

Comment:
Also see https://devnet.jetbrains.com/message/5532560#5532560, I think it's pretty good idea for convention.